action.yml

```diff
@@ -32,6 +35,7 @@ runs:
     - ${{ inputs.skipIfExists }}
     - ${{ inputs.variables }}
     - ${{ inputs.filename }}
+    - ${{ inputs.dependencies }}
```

Comment:
Could there be any issue in the order of parameters for users using before the log-level? I mean, that the log-level parameter might be used as `dependencies`?

Reply:
When a action is executed we rebuild the action and the inputs are calculated again, so there shouldn't be any problem